lib/vet360/person/service.rb

```diff
@@ -26,11 +33,21 @@ def init_vet360_id(icn)
       private
 
       # rubocop:disable Lint/UriEscapeUnescape
```

Comment:
I would recommend actually digging into this rubocop suggestion, it appears my example usage of `URI.escape` is deprecated and we should probably follow best practices.

https://stackoverflow.com/questions/2824126/whats-the-difference-between-uri-escape-and-cgi-escape